client/sites/marketing/connections/services/google-my-business.js

```diff
@@ -45,11 +45,12 @@ export class GoogleMyBusiness extends SharingService {
 		this.props
 			.connectGoogleMyBusinessLocation( this.props.siteId, keyringConnectionId, externalUserId )
 			.catch( () => {
-				this.props.failCreateConnection( {
-					message: this.props.translate( 'Error while linking your site to %(service)s.', {
+				this.props.warningNotice(
+					this.props.translate( 'Error while linking your site to %(service)s.', {
 						args: { service: this.props.service.label },
 					} ),
-				} );
+					{ id: 'publicize' }
```

Comment:
Not sure we need to keep the same ID for all of those, since the chance to see two different types of warnings on the same page sounds minimal, but I'll leave it to your judgment.

Reply:
I added the ID largely for consistency with other warning notices we're already showing for the same kind of issue:

https://github.com/Automattic/wp-calypso/blob/ee8dcfccfc2a4900a6bebb8004fc00617d620671/client/sites/marketing/connections/service.jsx#L198-L205

Reply:
Yes, the practical effect of the `id` is that you'll never see two notices with the same ID. The newer one will auto-remove the previous one. That sounds good.